open-company/open-company-web
# Responsive dashboard

cards:
- https://trello.com/c/iMPdOWak
- https://trello.com/c/iNCsK0mh
- https://trello.com/c/68DxJQ20
- https://trello.com/c/9g6RQnO1
- https://trello.com/c/glg8sNWt
- https://trello.com/c/kQuxcsn5
- https://trello.com/c/VkepRQV9

To test:
- deploy this branch (locally it would a problem since some resources like adobe fonts won't load using your local ip address)
- navigate to the home page
- select a company dashboard (the best is buffer as it has all kind of data)
- check the dashboard on your computer
- check the dashboard on your mobile device (iOS and android if possible)
- check the dashboard on a table if can
